README.md

```diff
@@ -1,235 +1,93 @@
-# 1Password Connect Python SDK
+<!-- Image sourced from https://blog.1password.com/introducing-secrets-automation/ -->
+<img alt="" role="img" src="https://blog.1password.com/posts/2021/secrets-automation-launch/header.svg"/>
 
-[![Python](https://img.shields.io/badge/python-3.7%20%7C%203.8%20%7C%203.9-blue)](https://www.python.org)
-[![PyPI version](https://badge.fury.io/py/onepasswordconnectsdk.svg)](https://badge.fury.io/py/onepasswordconnectsdk)
-![CI](https://github.com/1Password/connect-sdk-python/workflows/Test/badge.svg)
-[![License](https://img.shields.io/badge/license-MIT-blue.svg)](https://en.wikipedia.org/wiki/MIT_License)
-[![codecov](https://codecov.io/gh/1Password/connect-sdk-python/branch/main/graph/badge.svg?token=VBPCH0CU2E)](https://codecov.io/gh/1Password/connect-sdk-python)
+<div align="center">
+  <h1>1Password Connect SDK for Python</h1>
+  <p>Access your 1Password items in your Python applications through your self-hosted <a href="https://developer.1password.com/docs/connect">1Password Connect server</a>.</p>
+  <a href="#-get-started">
+    <img alt="Get started" src="https://user-images.githubusercontent.com/45081667/226940040-16d3684b-60f4-4d95-adb2-5757a8f1bc15.png" height="37"/>
+  </a>
+</div>
 
-The 1Password Connect SDK provides access to 1Password via [1Password Connect](https://support.1password.com/secrets-automation/) hosted in your infrastructure. The library is intended to be used by Python applications to simplify accessing items in 1Password vaults.
+---
 
-## Prerequisites
+The 1Password Connect SDK provides access to 1Password via [1Password Connect](https://developer.1password.com/docs/connect) hosted in your infrastructure. The library is intended to be used by Python applications to simplify accessing items in 1Password vaults.
 
-- [1Password Connect](https://support.1password.com/secrets-automation/#step-2-deploy-a-1password-connect-server) deployed in your infrastructure
-## Installation
+## 🪄 See it in action
 
-To install the 1Password Connect Python SDK:
-```bash
-$ pip install onepasswordconnectsdk
-```
+Check the [Python Connect SDK Example](example/README.md) to see an example of item manipulation using the SDK that you can execute on your machine.
 
-To install a specific release of the 1Password Connect Python SDK:
-```bash
-$ pip install onepasswordconnectsdk==1.0.1
-```
+## ✨ Get started
 
-## Usage
+1. Install the 1Password Connect Python SDK:
 
-**Import 1Password Connect Python SDK**
+   ```sh
+   pip install onepasswordconnectsdk
+   ```
 
-```python
-import onepasswordconnectsdk
-```
+2. Export the `OP_CONNECT_HOST` and `OP_CONNECT_TOKEN` environment variables:
 
-**Environment Variables**
+   ```sh
+   export OP_CONNECT_HOST=<your-connect-host> && \
+       export OP_CONNECT_TOKEN=<your-connect-token>
```

Comment:
Alignment is off, looks like some extra spaces were added in front of the `export` here.

```suggestion
   export OP_CONNECT_TOKEN=<your-connect-token>
```

Reply:
This is intentional. Multiline commands can have identation for the following reasons:
- Clear distinction that it's one single command instead of two distinct ones
- Shows nicely when passing multiple things to a single command (similar to calling a function in code and format it on multiple lines). 
 
We have examples like those in the Dev Portal as well such as the [example snippets](https://developer.1password.com/docs/cli/reference/management-commands/item/#item-create-examples) for `op item create`:

```sh
op item create --category=login --title='My Example Item' --vault='Test' \
    --url=https://www.acme.com/login --generate-password=20,letters,digits \
    username=jane@acme.com \
    'Test Field 1=my test secret' \
    'Test Section 1.Test Field2[text]=Jane Doe' \
    'Test Section 1.Test Field3[date]=1995-02-23' \
    'Test Section 2.Test Field4[text]='$myNotes
```

I do see your point that here the identation may not fully work here, since it's only two lines. 🤔